.github/CODE_OF_CONDUCT.md

```diff
@@ -17,23 +17,23 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for our
```

Comment:
Looking here - I'm thinking that everything here inside:
- `.github/`
- `README.md`
- `Scribe-i18n/`

can get removed probably. Looks like they really are from consolidating previous commits. 

Looks like the things that we'd want are what is under:
- `Scribe.xcodeproj/`
- `Scribe/`

as we can kinda see in the original commit for the PR, 80aa484d52f945335594d7589b87da07fd02839e